ml-agents/mlagents/trainers/tests/test_reward_signals.py

```diff
@@ -92,64 +117,127 @@ def create_ppo_policy_mock(
     return env, policy
 
 
-@mock.patch("mlagents.envs.UnityEnvironment")
-def test_curiosity_cc_evaluate(mock_env, dummy_config, curiosity_dummy_config):
-    env, policy = create_ppo_policy_mock(
-        mock_env, dummy_config, curiosity_dummy_config, False, False, False
-    )
+def reward_signal_eval(env, policy, reward_signal_name):
     brain_infos = env.reset()
     brain_info = brain_infos[env.brain_names[0]]
     next_brain_info = env.step()[env.brain_names[0]]
-    scaled_reward, unscaled_reward = policy.reward_signals["curiosity"].evaluate(
+    # Test evaluate
+    rsig_result = policy.reward_signals[reward_signal_name].evaluate(
         brain_info, next_brain_info
     )
-    assert scaled_reward.shape == (12,)
-    assert unscaled_reward.shape == (12,)
+    assert rsig_result.scaled_reward.shape == (12,)
+    assert rsig_result.unscaled_reward.shape == (12,)
+
+
+def reward_signal_update(env, policy, reward_signal_name):
+    brain_info_list = []
+    for i in range(20):
```

Comment:
There are a few numerical values hard-coded here. Can we replace them with vars/consts?